composer.json

```diff
@@ -58,6 +59,10 @@
     {
       "type": "git",
       "url": "https://github.com/civicrm/zetacomponents-mail.git"
+    },
+    {
+      "type": "git",
+      "url": "https://github.com/totten/topsort.php.git"
```

Comment:
Why use the fork and not the original?

Reply:
I believe it's forked because the original doesn't support 5.3.  However, we are deprecating 5.3 at the end of the year... If this is committed as is, it would be good to have somewhere to document workarounds required for 5.3 that can be removed when we drop 5.3.

Reply:
Quite right. Maybe we can file in JIRA and use an extra label to flag issues blocked on PHP 5.4? (eg `php54`?)